src/main/kotlin/net/atos/zac/search/SearchService.kt

```diff
@@ -133,16 +143,22 @@ class SearchService @Inject constructor(
 
     private fun getFilterQueriesForDatumsParameters(zoekParameters: ZoekParameters): List<String> =
         zoekParameters.datums.map { (dateField, date) ->
-            val from = date.van?.let { DateTimeFormatter.ISO_INSTANT.format(it.atStartOfDay(ZoneId.systemDefault())) } ?: "*"
-            val to = date.tot?.let { DateTimeFormatter.ISO_INSTANT.format(it.atStartOfDay(ZoneId.systemDefault())) } ?: "*"
+            val from = date.van?.let {
+                DateTimeFormatter.ISO_INSTANT.format(it.atStartOfDay(ZoneId.systemDefault()))
```

Comment:
```suggestion
               ISO_INSTANT.format(it.atStartOfDay(ZoneId.systemDefault()))
```

You could just import `DateTimeFormatter.ISO_INSTANT`.